src/components/ServicePage/ServicePage.scss

```diff
@@ -1,144 +1,144 @@
-@media only screen and (max-width:755px) {
-    .services-column {
-        max-width: 33% !important;
-    }
+// @media only screen and (max-width:755px) {
+//     .services-column {
+//         max-width: 33% !important;
+//     }
```

Comment:
Please Delete this file it's causing a merge conflict. Delete service.js as well please.

Reply:
Deleted the file, thanks!